dependencies.gradle

```diff
@@ -2,7 +2,7 @@
 
 dependencies {
     shadowImplementation('com.github.GTNewHorizons:AVRcore:master-SNAPSHOT')
-    compile('com.github.GTNewHorizons:GT5-Unofficial:5.09.41.00:dev')
+    compile('com.github.GTNewHorizons:GT5-Unofficial:5.09.41.02-pre:dev')
```

Comment:
Don't use pre versions in dependencies.gradle

Reply:
i add it for testing a gui but this is not longer necessary . Will set it to a new tag